aggsender/aggsender.go

```diff
@@ -170,7 +170,7 @@ func (a *AggSender) checkSendCertificateStopCondition(err error) {
 			a.log.Info("Stopping AggSender because StopOnFinishedSendingAllCertificates is true")
```

Comment:
It should be safe to remove this line, because we are relying on `Panicf` function.